src/App.tsx

```diff
@@ -75,7 +84,8 @@ const App = () => {
         <ThemeProvider theme={theme}>
           <CustomButton
             variant="outlined"
-            size="medium">
+            size="medium"
+            onClick={resetGame}>
```

Comment:
Let's add a confirm dialogue in case the user missclick this button.

Reply:
Okaay!